### PR TITLE
Devtools Tips 18: PWA

### LINKED ISSUE #4375: [devtools-tips] 18 - DevTools for PWA
Details: b/246353777
Deadline: Jan 26
Notes: Do not merge the PR when ready, tag @jecfish to merge in time with video publish
Also refresh relevant docs in d.c.c and web.dev as well.

### PR DESCRIPTION
**Pull request by sofiayem**
Fixes #4375. Publish today.

A promo post for DTT18: PWA.

The corresponding doc refresh is in progress.

### CODE REVIEW

File: site/en/blog/devtools-tips-18/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - sofiayem
+hero: 'image/NJdAV9UgKuN8AhoaPBquL7giZQo1/OEqDXbxeEqiGeZnKxr60.png'
+alt: >
+  DevTools Tips hero logo
+tags:
+  - devtools
+  - devtools-tips
+---
+
+Progressive Web Apps (PWA) are apps built and enhanced with modern APIs to deliver enhanced capabilities, reliability, and installability while reaching anyone, anywhere, on any device, all with a single codebase.
```
Comment (jecfish) [inline site/en/blog/devtools-tips-18/index.md:18]: Add relevant link for "Progressive Web Apps (PWA)"

Comment (jecfish) [reply site/en/blog/devtools-tips-18/index.md:18]:
```suggestion
Progressive Web Apps (PWA) are web apps built and enhanced with modern APIs to deliver enhanced capabilities, reliability, and installability while reaching anyone, anywhere, on any device, all with a single codebase.
```

**Review by jecfish — APPROVED** [review]
LGTM % comment.

A few notes:
1. I was hoping the Manifest pane documentation is up in time with the promo post.
2. Is our debugging PWA docs (https://goo.gle/devtools-pwa) at least include the features mentioned in the video? It is because we encourage developer to learn more from our docs: 
3. Have we update anywhere (web.dev / d.c.c) to include a link to our debugging PWA docs (e.g. web.dev/learn/pwa/)?

**Comment by sofiayem** [comment]
> LGTM % comment.
> 
> A few notes:
> 
> 1. I was hoping the Manifest pane documentation is up in time with the promo post.
> 2. Is our debugging PWA docs (https://goo.gle/devtools-pwa) at least include the features mentioned in the video? It is because we encourage developer to learn more from our docs:
> 3. Have we update anywhere (web.dev / d.c.c) to include a link to our debugging PWA docs (e.g. web.dev/learn/pwa/)?

1. Manifest is described and somewhat up-to-date. Unfortunately, not 100%.
2. Not all. It includes featured not mentioned in the video. So, the users can still learn more.
3. Not yet.

I'm working on it.

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/devtools-tips-18/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: >
+  DevTools Tips: Debugging PWA
+description: >
+  Debug Progressive Web Apps with DevTools.
+layout: 'layouts/blog-post.njk'
+date: 2023-01-26
```
Comment (sofiayem) [inline site/en/blog/devtools-tips-18/index.md:7]:
```suggestion
date: 2023-01-30
```

**Pull request merged**